### ISSUE TITLE
 Missing: libvips, libnotify (not required)

### ISSUE THREAD

**Issue by VampiricAlien** [sylviiu/ezytdl#39]
Appimage for v2.4.2 and v3.0.0-dev.135 say it's missing dependency but also says it's not required?
```
Starting ezytdl v2.4.2
getPath ./util/promisifiedFS (promise: false)
Verifying dependencies at /tmp/.mount_ezytdl8JbGQb/resources/app.asar/core/depcheck/linux
depchecks available for platform linux -- running
Checking dependency linux/libnotify.js
checking for libnotify | undefined
libnotify.js = false
Checking dependency linux/sharp.js
sharp.js = false
```

```
Starting ezytdl v3.0.0-dev.135 in mode "main"
getPath ./system/main.js (promise: false)
getPath ./util/promisifiedFS (promise: false)
Verifying dependencies at /tmp/.mount_ezytdljpFvMz/resources/app.asar/core/depcheck/linux
depchecks available for platform linux -- running
Checking dependency linux/libnotify.js
checking for libnotify | undefined
libnotify.js = false
Checking dependency linux/sharp.js
sharp.js = false
[ [ 'libvips' ] ] [ [ 'libnotify' ] ]
createdialog called but app not ready!
quitting (function called)
System platform linux; file name will be bridge-linux
App data location: /home/A21/.config/ezytdl
----------------------------------
CREATING NEW DOWNLOADMANAGER: default
----------------------------------
basepath = /tmp/.mount_ezytdljpFvMz/resources/
Process exiting with code 0
createDialog error Failed to finish startup process! Failed to start ezytdl v3.0.0-dev.135 -- your system is missing dependencies!

- Missing: libvips, libnotify (not required)
Loading took too long!

```

**Comment by sylviiu** [comment]
`libvips` is required, `libnotify` is not. i should probably make that a little more clear on the error message

**Comment by sylviiu** [comment]
I have NEVER seen that before... I'll look into that when I'm home.

**Comment by sylviiu** [comment]
can you provide all of the logs leading up to that point? and did you do anything to trigger that error?

**Comment by sylviiu** [comment]
If you're still trying the appimage, I'd suggest trying the tar.gz build -- appimage build has been having a lot of issues lately with speed where the tar.gz build doesn't.

**Comment by sylviiu** [comment]
tar.gz isn't an install either; you just extract it and run it in place. Linux currently has no installers (that I know of) for this app

**Comment by ArgX11** [comment]
Don't want to create a new issue so asking this question here, 
Will there be a win64.exe version in the future?

**Comment by sylviiu** [comment]
> That worked!
> 
> Question, where does the downloaded ffmpeg go? system never updated after download.

Downloaded FFmpeg goes in the config path (on linux i think it's `~/.config/ezytdl`) in its own folder (which should be `ffmpeg-linux`)

**Comment by sylviiu** [comment]
> Don't want to create a new issue so asking this question here, Will there be a win64.exe version in the future?

the app is 64 bit already, there are no 32 bit builds
![image](https://github.com/sylviiu/ezytdl/assets/95180094/b9da744f-cc9f-4372-bdf3-21a6740a64f8)

"win32" doesn't describe if it's a 32 bit or 64 bit application – in this case, it's simply the program API (i think it was called) used to run node.js(?) (i'm not too sure myself)
![image](https://github.com/sylviiu/ezytdl/assets/95180094/d00e26f0-3b0b-48bb-89d5-a4409b6388f1)

microsoft kept the name "win32" after 64 bit windows became a thing because "win64" is (to my knowledge) an extension of the win32 API

(you can also find this by using the console in the app itself, it is built for 64 bit)
![image](https://github.com/sylviiu/ezytdl/assets/95180094/7d276b67-dd93-49e6-8b0b-12e09e44f8c4)

**Comment by sylviiu** [comment]
(sidenote that since it worked i will close this issue, i'll try to figure out why appimage builds have become so painfully slow to run soon)

**Comment by sylviiu** [comment]
slow appimage speed has been fixed! (2185f11)

appimages used to build with maximum compression, which slowed them down LOTS (which previously I think didn't have issues??) -- they now build with the default compression, making them run much faster.

next dev build should let you use appimage again (which allows in-place auto updating)
